src/App.tsx

```diff
@@ -2,24 +1,40 @@
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const items = getNumbers(1, 42).map(n => `Item ${n}`);
+
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [perPage, setPerPage] = useState<number>(5);
```

Comment:
Consider using a single `useState` for pagination settings. It could be an object with `currentPage` and `perPage` properties. This way, you can update both values in a single call, which can be more efficient.